src/Tracker.jl

```diff
@@ -76,7 +76,17 @@ include("lib/array.jl")
 include("back.jl")
 include("numeric.jl")
 include("forward.jl")
-@init @require PDMats="90014a1f-27ba-587c-ab20-58faa44d9150" include("lib/pdmats.jl")
+
+if !isdefined(Base, :get_extension)
+  using Requires
+end
+
+@static if !isdefined(Base, :get_extension)
+function __init__()
```

Comment:
May I ask why this can't be one block with `@static if`?

Reply:
Basically due to:

```
julia> begin
       using Requires
       @requires foo
       end
ERROR: LoadError: UndefVarError: `@requires` not defined
in expression starting at REPL[1]:3
```

You cannot load a package defining a macro and call that macro in the same block.

Reply:
Ah ok, thanks!